src/mpid/ch4/src/ch4_recv.h

```diff
@@ -250,9 +266,10 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_recv_safe(void *buf,
         *(req) = MPIR_Request_create(MPIR_REQUEST_KIND__RECV);
         MPIR_ERR_CHKANDSTMT((*req) == NULL, mpi_errno, MPIX_ERR_NOREQ, goto fn_fail, "**nomemreq");
         MPIR_Datatype_add_ref_if_not_builtin(datatype);
-        MPIDI_workq_pt2pt_enqueue(RECV, NULL /*send_buf */ , buf, count, datatype,
-                                  rank, tag, comm, context_offset, av,
-                                  status, *req, NULL /*flag */ , NULL /*message */ ,
+        MPIDI_workq_pt2pt_enqueue(RECV, 0 /* transport */ , NULL /*send_buf */ , buf, count,
```

Comment:
Use enum value here for transport argument?

Reply:
Done